README.md

```diff
@@ -1,6 +1,6 @@
 ### Study Notes
 
-[git CLI cheatsheet](https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet)
+[git CLI cheatsheet](https://github.com/getfutureproof/lfa_guides_wiki/wiki/git-CLI-Cheatsheet)
```

Comment:
Can you please change this link back to `[git CLI cheatsheet](https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet)`

Reply:
Sorry to be a pain, but the resource does not exist - We have not updated the wiki yet, so until we do, the only way for them to have access the cheatsheet is via https://github.com/getfutureproof/fp_guides_wiki/wiki/git-CLI-Cheatsheet


![image](https://github.com/getfutureproof/lfa_study_notes_hello_github/assets/48137278/8cff4112-a410-4bc2-8d65-1307616ec367)

Reply:
True... the wiki is for the old curriculum though, so we will need to think about how to proceed on this. @TDG-Lafosse what do you think?